randovania/games/cave_story/patcher/caver_music_shuffle.py

```diff
@@ -153,6 +153,7 @@ def shuffle(self, rng: Random, cosmetic: CSCosmeticPatches) -> dict[CaverCue, di
     @classmethod
     def get_shuffled_mapping(cls, rng: Random, cosmetic: CSCosmeticPatches) -> dict[str, dict[str, str]]:
         music = cls.get_randomizer(cosmetic.music_rando.randomization_type)
+        # FIXME: The typing for this/return value seems to be very weird. `events` also gets redefined.
```

Comment:
Would appreciate help with this fixme. Would be nice to get it fixed before merging so that i can put in the pyproject.toml just randovania/games/cave_story